tests/testapp/tests.py

```diff
@@ -28,6 +28,17 @@ def test_validate_org_relation_pk_comparison_bug(self):
         c.save()
         self.assertEqual(Config.objects.count(), 1)
 
+    def test_validate_org_relation(self):
+        c = Config(name='test')
+        self.assertEqual(c._validate_org_relation(''), None)
```

Comment:
change this to the following which should be easier to understand for someone who has some familiarity with the code of this module:
```python
# simulates validating a relation instance attribute that has not been set yet
self.assertEqual(c._validate_org_relation('not_set_yet'), None)
```